include/NovelRT/Ecs/Configurator.h

```diff
@@ -21,6 +22,7 @@ namespace NovelRT::Ecs
         bool _shouldAddDefaultSystems = false;
         std::optional<uint32_t> _threadCount;
         std::vector<std::function<void(Timing::Timestamp, Catalogue)>> _systems;
+        std::vector<std::shared_ptr<IEcsSystem>> _iecsSystems;
```

Comment:
I do wonder of there's a better name for this collection but it does what it says on the tin and I can't think of anything better right now so best just leave it alone. Mostly just writing this for transparency in case other maintainers see this and have an idea.